<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>roomlist</class>
 <widget class="QDialog" name="roomlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>欢迎来到德州扑克</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../source/picture/icon/beauty.jpg</normaloff>../source/picture/icon/beauty.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#roomlist{border-image: url(:/source/picture/background/roomList.jpg);}</string>
  </property>
  <widget class="QPushButton" name="roomlist1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>240</width>
     <height>65</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/picture/background/button1.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="roomlist2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>240</width>
     <height>65</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/picture/background/button2.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="roomlist3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>240</width>
     <height>65</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/picture/background/button3.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="roomlist4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>240</width>
     <height>65</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/picture/background/button4.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="roomlist5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>600</y>
     <width>240</width>
     <height>65</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/picture/background/button5.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rank">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/picture/icon/rank.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="setting">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>60</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/picture/icon/setting.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tips">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>60</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/picture/icon/tips.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_rank">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>110</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();
color: rgb(255, 255, 255);
font: 75 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>排行</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_setting">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>110</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();
color: rgb(255, 255, 255);
font: 75 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tips">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>110</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();
color: rgb(255, 255, 255);
font: 75 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>提示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="person">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>55</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/picture/icon/person.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();
font: 75 12pt &quot;宋体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>我的名字</string>
   </property>
  </widget>
  <widget class="QLabel" name="usergold">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();
font: 75 10pt &quot;宋体&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>1000000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/picture/icon/chouma.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>92</y>
     <width>120</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();
background-color: rgb(85, 170, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="room_list" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>450</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pageup">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>260</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/picture/icon/pageup.png);
background-color: rgb(147, 98, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pagedown">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>500</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/picture/icon/pagedown.png);
background-color: rgb(147, 98, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="_currentpage">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>385</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
